### PR TITLE
MF Cleanup 3

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: apps/financiers/financiers/src/app/marketplace/organization/list/list.component.html [hunk 1]
```diff
@@ -35,16 +35,14 @@ <h1 class="mat-display-2 page-title">Our partners</h1>
     <org-card [org]="org"></org-card>
   </ng-template>
 
-  <ng-template listPageEmpty *ngIf="loading$.value">
+  <ng-template listPageEmpty *ngIf="loading$ | async">
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/organization/list/list.component.html:38]: `asObservable`?

File: apps/financiers/financiers/src/app/marketplace/organization/list/list.component.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Angular
-import { Component, ChangeDetectionStrategy, HostBinding, OnInit, OnDestroy } from '@angular/core';
+import { Component, ChangeDetectionStrategy, HostBinding, OnInit, OnDestroy, ChangeDetectorRef } from '@angular/core';
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/organization/list/list.component.ts:2]: why importing if you are not using it

File: apps/financiers/financiers/src/app/marketplace/title/list/list.component.html [hunk 3]
```diff
@@ -84,16 +84,14 @@ <h1 class="mat-display-2 page-title">All Projects</h1>
     </movie-card>
   </ng-template>
 
-  <ng-template listPageEmpty *ngIf="loading$.value">
+  <ng-template listPageEmpty *ngIf="loading$ | async">
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/title/list/list.component.html:87]: same above

File: libs/ui/src/lib/list/page/list-page.component.ts [hunk 4]
```diff
@@ -80,7 +80,7 @@ export class ListPageComponent implements AfterContentInit {
   template: `
   <ng-content></ng-content>
   <mat-progress-bar color="primary" [value]="value"></mat-progress-bar>
-  <button mat-stroked-button color="primary" (click)="loadMore.emit()" [disabled]="value === 100 || !value">LOAD
+  <button mat-stroked-button color="primary" (click)="loadMore.emit()" *ngIf="value === 100 || !value">LOAD
```
Comment (fritzschoff) [inline libs/ui/src/lib/list/page/list-page.component.ts:83]: you checked with design about that?

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: apps/financiers/financiers/src/app/marketplace/organization/list/list.component.html (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/financiers/financiers/src/app/marketplace/organization/list/list.component.html:38]: no need really. I know it's recommended by the angular team, but the asObservable is just a security lock for public API to be sure the consumer is not using the writing method on the observable

File: libs/ui/src/lib/list/page/list-page.component.ts (hunk 4 above)
Comment (GrandSchtroumpf) [reply libs/ui/src/lib/list/page/list-page.component.ts:83]: yep

File: apps/financiers/financiers/src/app/marketplace/title/list/list.component.html (hunk 3 above)
Comment (GrandSchtroumpf) [reply apps/financiers/financiers/src/app/marketplace/title/list/list.component.html:87]: same answer as above

File: apps/financiers/financiers/src/app/marketplace/organization/list/list.component.html (hunk 1 above)
Comment (fritzschoff) [reply apps/financiers/financiers/src/app/marketplace/organization/list/list.component.html:38]: alright, sounds good

**Review by fritzschoff — APPROVED** [review]

**Pull request merged**